### PR TITLE
fix(answer): prevent recalculation of calculated answers for default answer changes

### PR DESCRIPTION
**Pull request by anehx**

### CODE REVIEW

File: caluma/caluma_form/tests/test_answer.py [hunk 1]
```diff
@@ -256,6 +256,35 @@ def test_save_default_answer_python_api(
...
+def test_save_calculated_dependency_default_answer(
+    db,
+    snapshot,
+    question,
+    answer,
+    question_option,
+    document_factory,
+    answer_factory,
+    answer_document_factory,
+    question_factory,
```
Comment (sbor23) [inline caluma/caluma_form/tests/test_answer.py:269]: There are a couple of unused fixture / factories used here.

**Review by sbor23 — COMMENTED** [review]
Is this a fix or an optimization? Did the current behaviour trigger any unexpected outcome?
If so, a test case which would trigger the bug would be nice.

**Comment by anehx** [comment]
> Is this a fix or an optimization? Did the current behaviour trigger any unexpected outcome?
> If so, a test case which would trigger the bug would be nice.

It's a fix and there is a test.....

File: caluma/caluma_form/tests/test_answer.py (hunk 1 above)
Comment (anehx) [reply caluma/caluma_form/tests/test_answer.py:265]: removed

**Review by sbor23 — APPROVED** [review]
LGTM

**Pull request merged**